.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+// For format details, see https://aka.ms/devcontainer.json
+{
+	"name": "pallets/flask",
+
+	// Features to add to the devcontainer
+	"features": {
+		"ghcr.io/devcontainers/features/python:1": {}
+	},
+
+	// Displays a custom welcome message
+	"onCreateCommand": "sudo cp .devcontainer/welcome-message.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt",
+
+	// Runs the initial setup commands after the container is created.
+	"postCreateCommand": "sudo chmod +x .devcontainer/post-create-command.sh && .devcontainer/post-create-command.sh"
```

Comment:
This only runs at initial creation or if users rebuild their codespace. Stopping and restarting a codespace will put users back where they left off.